proto/dydxprotocol/ratelimit/query.proto

```diff
@@ -23,6 +24,12 @@ service Query {
     option (google.api.http).get =
         "/dydxprotocol/v4/ratelimit/capacity_by_denom";
   }
+  // Get all pending send packets
+  rpc GetAllPendingSendPackets(GetAllPendingSendPacketsRequest)
```

Comment:
Nit: conventionally, we name rpc methods without "Query" or "Get", but prefix the request object name with "Query". See [example](https://github.com/dydxprotocol/v4-chain/blob/d7333d4dbba60f20b6225d8ef788282775894e81/proto/dydxprotocol/perpetuals/query.proto#L20)